### DISCUSSION TITLE
ZB-ONOFFPlug-D0005 - Not reporting power value

### DISCUSSION THREAD

**Discussion by McFarTech** [Koenkk/zigbee2mqtt#15727] [Devices]
Hello. 

I am running Zigbee2MQTT under Home Assistant. I am running  version: 1.28.4-1 and my HA is running

Home Assistant 2022.12.8
Supervisor 2022.11.2
Operating System 9.4
Frontend 20221213.1 - latest

I run a bunch of Samsung Smart things plugs and they are all reporting their Power and Energy values. I have this one plug that is not reporting its Power value and its stopped reporting its energy value as well. 

The plug functions as it should , on / off etc. The plug details are below and this is the same as all my other SmartThings plugs which are reporting ok. 

Firmware build date
20190716-165
Firmware version
2.3
Manufacturer
[SmartThings](https://www.zigbee2mqtt.io/supported-devices/#v=SmartThings)
Model
[GP-WOU019BBDWG](https://www.zigbee2mqtt.io/devices/GP-WOU019BBDWG.html#smartthings-gp-wou019bbdwg)

Are there any logs or can anyone help get this reporting again?

Thanks in advance for any help.

**Comment by jackwilsdon** [comment]
I was having this same issue with the exact same plug - performing a reconfigure of the device looks to have fixed it.

**Comment by Skynetnexus** [comment]
I am also have this issue after upgrading to 1.37.1-dev commit: 9b3a394. The plugs were previously reporting energy but now one that I reset and re-paired to HA reports "null" and the other one has it's energy value stalled at 46.79.

EDIT :- Downgraded to an earlier version ( by restoring Z2M only from a full backup ) and my 2 outlets are back to reporting energy values. BTW my outlets are being picked up as GP-WOU019BBDWG Firmware 20200103-586  version 2.6
